pkg/asset/installconfig/powervs/platform.go

```diff
@@ -0,0 +1,128 @@
+package powervs
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+
+	survey "github.com/AlecAivazis/survey/v2"
+	"github.com/AlecAivazis/survey/v2/core"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+
+	"github.com/openshift/installer/pkg/types/powervs"
+)
+
+// Platform collects powervs-specific configuration.
+func Platform() (*powervs.Platform, error) {
+	regions := knownRegions()
+	
+        // TODO(cklokman): This section came from aws and transforms the response from knownRegions
+        //                 into long and short regions to prompt the user for region select this section
+        //                 need need to be different based on powervs's implementation of knownRegions
+        //
+
+        longRegions := make([]string, 0, len(regions))
+	shortRegions := make([]string, 0, len(regions))
+	for id, location := range regions {
+		longRegions = append(longRegions, fmt.Sprintf("%s (%s)", id, location))
+		shortRegions = append(shortRegions, id)
+	}
+        
+	var regionTransform survey.Transformer = func(ans interface{}) interface{} {
+		switch v := ans.(type) {
+		case core.OptionAnswer:
+			return core.OptionAnswer{Value: strings.SplitN(v.Value, " ", 2)[0], Index: v.Index}
+		case string:
+			return strings.SplitN(v, " ", 2)[0]
+		}
+		return ""
+	}
+
+        // TODO(cklokman): We need to verify that us_south is the correct defaultRegion to use
+        //
+	defaultRegion := "us_south"
+	if !IsKnownRegion(defaultRegion) {
+		panic(fmt.Sprintf("installer bug: invalid default powervs region %q", defaultRegion))
```

Comment:
instead of panic() can you return an error?

Reply:
I can, however aws panics and would prefer to be consistent.  This should only ever pop up when the hard coded default region here is not located in the repo that is being used to acquire this data.  

Ultimately powervs will be handling this differently and the function IsKnownRegion will be different as a result.  All of that said... if we run with your other comment and remove defaultRegion all together, then this check isn't required anyway.  

Is there a reason we would prefer to return an error instead of panic()?

Reply:
as per our conversation this morning, this is fine. thanks for the context